test/zepto.html

```diff
@@ -824,10 +824,15 @@
       },
 
       testReady: function(t){
-        t.assertEqual('hi!', globalVarSetFromReady)
-        t.assertEqual('hi!', globalVarSetFromReady2)
-        t.assertEqual('hi!', globalVarSetFromReady3)
-        t.assertEqual('hi!', globalVarSetFromReady4)
+        t.pause()
+        setTimeout(function(){
+          t.resume(function(){
+            t.assertEqual('hi!', globalVarSetFromReady)
+            t.assertEqual('hi!', globalVarSetFromReady2)
+            t.assertEqual('hi!', globalVarSetFromReady3)
+            t.assertEqual('hi!', globalVarSetFromReady4)
+          })
+        }, 500)
```

Comment:
It works for me w/o `setTimeout`. I guess 'load' should occurs after 'DOMContentLoaded' in any cases, so it’s really redundant.